src/Common/src/Interop/Windows/sspicli/SecurityPackageInfoClass.cs

```diff
@@ -17,12 +17,10 @@ internal class SecurityPackageInfoClass
         internal string Comment = null;
 
         /*
-         *  This is to support SSL under semi trusted environment.
-         *  Note that it is only for SSL with no client cert
-         *
-         *  Important: safeHandle should not be Disposed during construction of this object.
-         * 
-         *  _SecPkgInfoW in sspi.h
+            This is to support SSL under semi trusted environment (only for SSL with no client cert).
```

Comment:
>This is to support SSL under semi trusted environment

What does "semi trusted" mean? Partial-trust vs. Full trust, i.e. ASP.NET environments using .NET Framework (Desktop/GAC)?  Does this still apply to CoreFx?